### PR TITLE
 Clone apache spark source code to parse the git commit IDs

### PR DESCRIPTION
**Pull request by NvTimLiu**
Clone apache spark source code to parse the git commit IDs

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: scripts/audit-spark-3.2.sh [hunk 1]
```diff
@@ -42,30 +42,28 @@ do
...
-  exit 1
-else
-  if [ -f "$lastcommit" ]; then
-      cd ${SPARK_TREE}
-      latestcommit=`cat ${lastcommit}`
-      git checkout $basebranch
-      git log --oneline HEAD...$latestcommit -- sql/core/src/main sql/catalyst/src/main | tee $COMMIT_DIFF_LOG
-      git log HEAD -n 1 --pretty="%h" > ${lastcommit}
+SPARK_TREE="$WORKSPACE/spark-3.0/spark"
+rm -rf $SPARK_TREE && git clone https://github.com/apache/spark.git $SPARK_TREE
```
Comment (NvTimLiu) [inline scripts/audit-spark-3.2.sh:46]: Here we clone spark source, to parse the git commit info

**Review by nartal1 — APPROVED** [review]
Thanks @NvTimLiu !

File: scripts/audit-spark-3.2.sh [hunk 2]
```diff
@@ -42,30 +42,28 @@ do
   esac
 done
 
-SPARK_TREE=$SPARK_HOME
-if [ -z "$SPARK_TREE" ]; then
-  echo "SPARK_HOME IS NOT SET. CANNOT RUN THIS SCRIPT"
-  exit 1
-else
-  if [ -f "$lastcommit" ]; then
```
Comment (NvTimLiu) [inline scripts/audit-spark-3.2.sh:50]: Only changed code indentations

File: scripts/audit-spark-3.2.sh [hunk 3]
```diff
@@ -42,30 +42,28 @@ do
...
-      cd ${SPARK_TREE}
-      latestcommit=`cat ${lastcommit}`
-      git checkout $basebranch
-      git log --oneline HEAD...$latestcommit -- sql/core/src/main sql/catalyst/src/main | tee $COMMIT_DIFF_LOG
-      git log HEAD -n 1 --pretty="%h" > ${lastcommit}
+SPARK_TREE="$WORKSPACE/spark-3.0/spark"
+rm -rf $SPARK_TREE && git clone https://github.com/apache/spark.git $SPARK_TREE
 
-      cd $WORKSPACE
```
Comment (NvTimLiu) [inline scripts/audit-spark-3.2.sh:57]: Only changed code indentations

**Pull request merged**